Makefile

```diff
@@ -265,6 +265,7 @@ DEFINES += ${CONFIG_DEFS:%=-D%}
 
 ifdef DEBUG
 DEFINES += -DDEBUG
+DEFINES += -DCONFIG_DEBUG_BUILD
```

Comment:
Can I ask what your use case is that you are wanting to do standalone debug kernel builds? Typically the only reason to build the kernel standalone is for verification, which is not a debug a build. Not saying we shouldn't fix this support, just curious as to your use case.